### DISCUSSION TITLE
List of issues in Win11 22557.1 with 22000.493.42.15 (latest)

### DISCUSSION THREAD

**Discussion by fpqc** [valinet/ExplorerPatcher#877] [General]
I know it's not officially supported yet, but I installed the EP nightly (22000.493.42.15) on the latest Win11 insider build (22557.1), and it is largely functional, save for the following few issues: 

1.) Right-clicking unoccupied regions of the taskbar doesn't bring up a context menu after a while. This is only really an issue when setting up something like a quick-launch bar. Restarting explorer.exe fixes the problem temporarily.

2.) There is a glitch in window previews where the preview is shown maybe too low, or the thumbnail isn't rendering, can't really tell for sure, but here's a picture:
![image](https://user-images.githubusercontent.com/6083371/154644656-836df4f4-cc7a-4afe-9994-ed60c8fdc169.png)

3.) Adjusting the volume with media keys crashes explorer.exe.  Manually using the slider through the volume flyout or settings menu does not crash explorer, though.

**Comment by bitcrusherrr** [comment]
I can confirm that 2nd issue occurs also on a non nightly EP (22000.469.41.16) and Win11 build 22557.

**Comment by highlanderc** [comment]
Preview is just showing behind the taskbar. I am not 100% sure this can be fixed as I believe the way explorer patcher works is that it overlays over the normal taskbar and since its a layer higher you can't click on the bottom taskbar.

**Reply by valinet** [reply]
Why do you have to “believe” things regarding ExplorerPatcher when all its source code (all its knowledge basically) is publicly available? What bottom taskbar and what preview? Sorry, but I don’t understand.

**Comment by forkicks** [comment]
I can confirm that running Windows 10 taskbar option in ExplorerPatcher in recent builds breaks the window thumbnails when you hover an active program icon in the taskbar. It just shows the name of the program and the actual thumbnail does not show (as in the screenshot on the OP) - this was working before.

**Comment by forkiesassds** [comment]
The only thing I can reproduce is broken window thumbnails using Windows 10 taskbar under 22H2 build 22567.

**Reply by valinet** [reply]
Yeah, that's indeed an issue.
